integration/integration_test.go

```diff
@@ -1603,7 +1623,7 @@ RUN date --utc > /root/date.txt`, testImageAlpine),
 			envbuilderEnv("CACHE_REPO", testRepo),
 			envbuilderEnv("GET_CACHED_IMAGE", "1"),
 		}})
-		require.ErrorContains(t, err, "uncached COPY command is not supported in cache probe mode")
+		require.Regexp(t, `uncached.*?command.*?is not supported in cache probe mode`, err.Error())
```

Comment:
```suggestion
		require.Regexp(t, `uncached.*command.*is not supported in cache probe mode`, err.Error())
```
The `?` is superfluous as `*` already allows zero or more.

Reply:
It kind of is, but it also makes the match non-greedy. This is just a reflexive thing for me as I find it makes the expression less likely to accidentally consume all of the input.